cognite/client/_api_client.py

```diff
@@ -483,12 +492,24 @@ def _list_generator(
                     else:
                         unprocessed_items = []
                     for chunk in chunks:
-                        yield list_cls._load(chunk, cognite_client=self._cognite_client)
+                        if list_cls._support_dict_load:
+                            yield list_cls._load(
+                                {"items": chunk, **first_response},  # type: ignore[dict-item]
```

Comment:
I think that assuming the other top-level keys always are static across responses - and thus only the first is needed - is not an assumption we can make here. (For type info it obviously works perfectly 😄 )